### ISSUE TITLE
disqus_identifier not being set

### ISSUE THREAD

**Issue by int-ua** [arthurk/django-disqus#78]
**Update/Resolution**: In case `{% with %}` is used `{% disqus_show_comments %}` must be placed before the closing `{% endwith %}`. Looks like it's worth a mention in the docs. Close it if you think otherwise.

**Older comment**:
This line does nothing:
https://github.com/arthurk/django-disqus/blob/0db52c240906c6663189c0a7aca9979a0db004d1/disqus/templates/disqus/show_comments.html#L6

Django 1.8.18
django-disqus 0.5
Python 3.4

I've set it up long ago and didn't check it. It was working without this config. At least we thought it was until we found out that comments are published on wrong pages and wrong links were formed by Disqus.

**Comment by int-ua** [comment]
Looks like #73 duplicate.

**Comment by int-ua** [comment]
With this code it outputs only `disqus_url`:

```
{% with item|model_name as disqus_prefix %}
{% with item.id|stringformat:"s" as item_id %}
	{% set_disqus_identifier disqus_prefix "_" item_id %}
{% endwith %}
{% endwith %}

{% set_disqus_url request.build_absolute_uri %}
```

**Comment by int-ua** [comment]
It is being set in `set_disqus_identifier()` but it's somehow no longer present in `get_config()`

**Comment by int-ua** [comment]
Found it. In case `{% with %}` is used `{% disqus_show_comments %}` must be placed before the closing `{% endwith %}`.

**Comment by SalahAdDin** [comment]
@int-ua Did you solve your problem?

Solition/acclaration must be putted in documentation.

**Comment by int-ua** [comment]
Yes, it should, but I'm not sure how to phrase it better, I'd like someone else doing it.

**Comment by SalahAdDin** [comment]
Let me see the documentation about it.

@int-ua Can you put here the final code?

And, can you sahre with me your user case? You are trying a list of post with number comments, aren you? Interesting.

I used the next code:
```html
<div id="fb-root">
        <!-- TODO: Review this with documentation -->
        {% set_disqus_identifier self.slug %}
        {% set_disqus_title self.title %}
        {% set_disqus_url self.full_url %}
        {% disqus_dev %}
        {% disqus_show_comments %}
    </div>
```

Is the normal use, for this reason there aren't other notes about other alternatives about how use it.

**Comment by int-ua** [comment]
My final code:

		{% with item|model_name as disqus_prefix %}
		{% with item.id|stringformat:"s" as item_id %}
			{% set_disqus_identifier disqus_prefix "_" item_id %}
			{% set_disqus_url request.build_absolute_uri %}
			{% disqus_show_comments %}
		{% endwith %}
		{% endwith %}

And I still get all comments on one page but that's another issue.
